sale_order_general_discount/models/sale.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+from lxml import etree
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    general_discount = fields.Float(
+        digits=dp.get_precision("Discount"), string="Discount (%)"
```

Comment:
get_precision method is deprecated... 
See --> https://github.com/odoo/odoo/blob/1a315e87b05fcbdefcb3db6d8d4837f6cc3e835c/odoo/addons/base/models/decimal_precision.py#L13